### ISSUE TITLE
Parametrizar link do audio no CLI

### ISSUE THREAD

**Issue by JoaoGFarias** [haskellcamargo/gemidao-do-zap#23]
Assim, você vai poder mandar qualquer aúdio, tornando a pegadinha praticamente infinita.
Nunca telefones tocarão tanto.

THE MONSTER IS LOOSE!!!
![mkd3284](https://user-images.githubusercontent.com/1585655/29100075-a840e074-7c7f-11e7-8744-abba68ff50fb.png)

**Comment by StanleySathler** [comment]
A ideia é boa, mas aí o repositório deixa de ser "gemidão-do-zap" :stuck_out_tongue_winking_eye:

**Comment by sayhicoelho** [comment]
Só você alterar no código o link do .mp3

**Comment by williamokano** [comment]
Concordo com o @StanleySathler, mas vai do autor querer isso @haskellcamargo

**Comment by williamthiago** [comment]
Tomara que os golpistas de sequestro falso não descubram isso... já pensou?

**Comment by williamokano** [comment]
`<sarcasm>`
É mesmo, ia ser foda eles mandando gemidao pros outros
`</sarcasm>`

**Comment by iujisato** [comment]
@williamokano  o que ia ser foda é eles utilizarem o serviço da totalvoice para simular chamadas com números de telefones de parentes das vítimas

Eu acho que o gemidao do zap, mesmo na brincadeira, escancarou uma falta de segurança absurda que existe no serviço telefônico

inclusive eu escrevi um bash script simples em que eu mandava uma lista de telefones (dos meus amigos) e o serviço fazia a ligação. O remetente era então sorteado randomicamente...
Imagina o caos no grupo, os caras falando 
"vc me ligou pra mandar gemidao do zap" 
"eu não, quem me ligou foi o fulano"
"eu não liguei pra ng, vc que me ligou"

**Comment by williamokano** [comment]
Peraí, editei minha mensagem anterior, agora fica melhor pra entender rsrs

**Comment by andrucz** [comment]
Poderia mover o código que faz a ligação pra outro projeto (que permitiria escolher o arquivo) e ajustar esse do gemidão pra usá-lo.

**Comment by jvhti** [comment]
Seria interessante se pusesse passar um áudio antes do gemidão (por exemplo, uma fala bem baixo, bom dia/boa tarde/boa noite, etc)... Faz parte da essência do gemidão enganar a pessoa!
